taskhawk/backends/aws.py

```diff
@@ -173,9 +173,11 @@ def process_message(self, queue_message) -> None:
     def delete_message(self, queue_message) -> None:
         queue_message.delete()
 
-    def nack_message(self, queue_message) -> None:
+    def nack_message(self, queue_message, visibility_s: int = 0) -> None:
```

Comment:
I wouldn't change this method, just change the behavior of `DelayedRetryException` (or `RetryException` with custom timeout) handling just extends visbility timeout and doesn't nack.